src/scripts/cliAMC.ts

```diff
@@ -42,23 +51,31 @@ async function main() {
                 const amcProductDataRepository = new AMCProductDataDefaultRepository(api);
                 const amcSubstanceDataRepository = new AMCSubstanceDataDefaultRepository(api);
                 const atcRepository = new GlassATCDefaultRepository(dataStoreClient);
+                const glassModuleRepository = new GlassModuleDefaultRepository(dataStoreClient);
 
                 try {
                     await setupLogger(instance, { isDebug: args.debug });
-                    logger.info(`Starting AMC recalculations...`);
+                    logger.info(`[${new Date().toISOString()}] Starting AMC recalculations...`);
                     const recalculateDataInfo = await getRecalculateDataInfo(atcRepository);
+                    const glassModule = await getGetAMCModuleById(glassModuleRepository, AMC_MODULE_ID);
+
                     logger.debug(
-                        `Recalculate data info: date=${recalculateDataInfo?.date}, recalculate=${
-                            recalculateDataInfo?.recalculate
-                        }, periods=${recalculateDataInfo?.periods.join(
+                        `[${new Date().toISOString()}] Recalculate data info: date=${
+                            recalculateDataInfo?.date
+                        }, recalculate=${recalculateDataInfo?.recalculate}, periods=${recalculateDataInfo?.periods.join(
                             ","
                         )} and orgUnitsIds=${recalculateDataInfo?.orgUnitsIds.join(",")}`
                     );
-                    if (recalculateDataInfo && recalculateDataInfo.recalculate) {
-                        logger.info(`Disabling AMC recalculations before start with calculations`);
+
+                    if (recalculateDataInfo) {
+                        logger.info(
+                            `[${new Date().toISOString()}] Disabling AMC recalculations before start with calculations`
```

Comment:
This logging pattern is duplicate a lot. Better if is extracted in a function:

```typescript
function logInfo(content: string) {
    logger.info(`[${new Date().toISOString()}] ${content}`);
}
```

Or similar